api/src/utils/media/csv/validation/csv-row-validator.ts

```diff
@@ -377,8 +406,16 @@ export const getUniqueColumnsValidator = (config?: FileColumnUniqueValidatorConf
 
     rows.forEach((row, rowIndex) => {
       const key = config.file_column_unique_validator.column_names
-        .map((columnIndex) => `${row[columnIndex] || ''}`.trim().toLowerCase())
+        .map((columnName) => `${row[columnName] || ''}`.trim().toLowerCase())
+        .filter(Boolean)
         .join(', ');
+
+      if (!key) {
+        // If key is empty, ignore this check.
+        // It is the job of a separate validation rule to ensure non-empty (required) cells
+        return;
+      }
+
```

Comment:
Addresses an issue where empty rows were being processed, and because an `<empty key> === `<empty key>` it was throwing errors when it shouldn't have.